alacritty/src/cli.rs

```diff
@@ -122,7 +124,9 @@ impl Options {
             .arg(
                 Arg::with_name("class")
                     .long("class")
+                    .value_name("instance> | <instance>,<general")
```

Comment:
```suggestion
                    .value_name("instance> | <instance>,<general>")
```

Reply:
This is intentional, it's necessary since clap adds `<...>` around the value by itself with no way to turn that off as far as I'm aware. This is the best solution I could find (it's also why the `<` is missing at the start).